tasks.py

```diff
@@ -1466,10 +1468,9 @@ def generate_deepcopy():
 
 
 def _align_helm_crds(source, output):
+    fetch_yq()
     run(
-        """yq eval-all 'select(.kind == "CustomResourceDefinition")' {} > {}""".format(
-            source, output
-        )
+        f"""{build_path}/bin/yq eval-all 'select(.kind == "CustomResourceDefinition")' {source} > {output}"""
```

Comment:
nit: you can use {} and yq_path directly here.

Reply:
Addressed the nit, and force-pushed.

I see other places where your comment applies, but I can fix them in a follow-up PR.